src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -837,7 +837,7 @@ void CoilCoolingDX::simulate(EnergyPlusData &state,
             Real64 dummyPLR = 1.0;
             int dummySpeedNum = 1;
             Real64 dummySpeedRatio = 1.0;
-            int dummyFanOpMode = 1.0;
+            HVAC::FanOp dummyFanOp = HVAC::FanOp::Cycling;
```

Comment:
Interesting use of dummy nodes to report out a coil size. Does using the actual coil nodes cause diffs?

Reply:
I didn't write this code. Just changed from int to enumeration.